mail_layout_preview/tests/test_layout_preview.py

```diff
@@ -61,10 +61,13 @@ def test_controller1(self):
         tree = etree.fromstring(content)
         list_items = tree.xpath("//ol[@class='email-template-list']/li/a")
         templates = self.env["mail.template"].search([("model_id.model", "=", model)])
-        url_pattern = "/email-preview/res.partner/mail.email_template_partner/{}"
-        for el, tmpl in zip(list_items, templates):
+        url_pattern = "/email-preview/res.partner/{templ_id}"
+        for el, tmpl in zip(list_items, templates, strict=False):
```

Comment:
isn't `False` the default value for `strict` ? 
If so, I see no point for this line diff